dockers/actions-plan-preview/planpreview.go

```diff
@@ -208,6 +235,14 @@ func makeCommentBody(event *githubEvent, r *PlanPreviewResult) string {
 			if app.ApplicationKind == "TERRAFORM" {
 				lang = "hcl"
 			}
+
+			l := utf8.RuneCountInString(app.PlanDetails)
+			if detailLen+int64(l) > reservedDetailMessagesLen {
+				fmt.Fprint(&b, detailsOmittedMessage)
+				detailLen += int64(utf8.RuneCountInString(detailsOmittedMessage))
+				continue
+			}
+			detailLen += int64(l)
```

Comment:
There is no diff in the failure application list so to be simple we don't have to care about them.

Reply:
fixed in [88266b1](https://github.com/pipe-cd/pipecd/pull/3253/commits/88266b199c4799088301b0374fcf7f8a131c27ce). PTAL